core/test-runtime/src/lib.rs

```diff
@@ -477,6 +516,36 @@ cfg_if! {
 					panic!("Failing because we are on wasm")
 				}
 
+				fn use_trie() -> u64 {
+					let pairs = [
```

Comment:
Both versions provide the same functionality, please move into one common function and just call this.

And you also don't call these functions somewhere in a test?